### PR TITLE
Display tags in `neuro ps`

### PR DESCRIPTION
**Pull request by ayushkovskiy**
Optionally: only if `.neuro.toml` config has explicit `tags` section.

Closes https://github.com/neuromation/platform-client-python/issues/1406

### CODE REVIEW

File: neuromation/cli/topics.py [hunk 1]
```diff
@@ -111,10 +111,10 @@ def topics() -> None:
       for indication of alignment, minimum and maximum column width,
       and optional column title:
 
-        `{id;align=center;min=10;max=30;width=20;ID TITLE}`
+        `{id;align=center;min=10;max=30;width=20;TITLE}`
```
Comment (ayushkovskiy) [inline neuromation/cli/topics.py]: Side change: `ID TITLE` (with space) is confusing.

**Comment by ayushkovskiy** [comment]
I didn't find any test on `neuro ps` output, so no new test for current functionality provided. Does this PR need a changelog note?

**Comment by serhiy-storchaka** [comment]
Yes, I think it needs a changelog note. Otherwise how users can know about it?

**Comment by ayushkovskiy** [comment]
Added changelog.

**Pull request merged**